### ISSUE TITLE
Open data

### ISSUE THREAD

**Issue by jvcasillas** [fosil-project/fosil-project.github.io#6]

**Comment by jvcasillas** [comment]
important references on transparency/open science practices papers: 

- psych: (get from ling one)
- ling: https://osf.io/preprints/metaarxiv/rcews/

**Comment by IvanAndreuRascon** [comment]
https://docs.google.com/document/d/1pv2Q3a922dwHabKtADTBggeyYMayY4p6uNwOoTHGq3w/edit?usp=sharing

**Comment by jvcasillas** [comment]
The recent reproducibility crisis in fields ranging from linguistics to biology underscores the urgent need for open data. 
Such a crisis arises when many studies cannot be replicated due to a lack of accessible supporting materials. 
To maintain ethical standards and ensure research outcomes can be built upon, major academic journals now mandate the provision of raw data. 
In linguistics, especially, open data is not just about transparency—it's about fostering trust, promoting cross-disciplinary collaboration, and democratizing knowledge. 
Embracing open data is, thus, essential for researchers aiming to contribute effectively to the discipline and the broader scientific community.

# The importance of Open data

1. Transparency: The world of linguistics, like any other scientific domain, needs transparent research. 
Open data paves the way for this transparency. 
It not only offers a clear window into the datasets but also ensures that the methodologies and results derived from these datasets are open for scrutiny. 
By doing so, it enables researchers and enthusiasts alike to understand, critique, and validate the vast expanse of linguistic studies.
2. Collaboration: Language, in its very essence, is a tool for communication and collaboration. 
It is only fitting that the study of language embodies these principles as well. 
Open data acts as a conduit for interdisciplinary projects. 
It opens the field-specific limitations, enabling global collaboration among linguists, thereby enriching the research outcomes with diverse perspectives.
3. Replicability: One of the cornerstones of robust research is the ability for its findings to be verified and replicated by independent entities. Open data ensures this replicability. 
With the data at their fingertips, researchers can cross-check findings, delve deeper into the datasets, and reaffirm the results, solidifying the integrity of the research.
4. Accelerating Research: Researches should be servants to society and progress.
A way of helping progress is helping other research teams. 
Gathering data can often be a painstaking and time-consuming process. 
Open data provides a solution to this bottleneck. 
By giving researchers direct access to expansive datasets, it drastically reduces the time and resources traditionally invested in data collection. 


# Sources of Open Linguistic Data. 


```{r}
 #| label: fig-open-science-benefits
#| echo: false
#| fig.cap: This is a caption
knitr::include_graphics("open_science.png")
```


## Why make your data publicly available?

In the digital age, traditional linguistic research methods are evolving and placing a stronger emphasis on open and easily accessible data for everyone.  
The change affects individual researchers, the academic community, and society.

**The Imperative of Data Sharing:**

- **For Researchers:** Sharing data elevates a researcher's academic profile. By doing so:
  * They gain enhanced **visibility**, positioning themselves prominently in an evolving academic landscape.
  * They can tap into a network of like-minded professionals, triggering **collaborations**.
  * They witness augmented **credibility** and **discoverability**, with their work now becoming a reference point for peers.
- **For the Research Community:** Open data serves as a catalyst for:
  * **Training** the next generation of researchers, offering them rich and better datasets for exploration.
  * Avoiding **redundancies** in experiments, optimizing resource utilization.
  * Ensuring robust **archiving** practices, preserving research for posterity.
  * And critically, facilitating **replication**, a key point of verifiable research.
- **For Society:** Publicly available data has broader societal ramifications. It:
  * Educates the **public**, bridging the gap between complex linguistic phenomena and general understanding.
  * Fuels **innovation**, with businesses and tech sectors leveraging linguistic insights.
  * Ensures **democratized access** to research, fostering a knowledge-driven society.

As linguistics recalibrates its approach to open science, there is a collective push towards increasing transparency. 
Initiatives worldwide now prioritize the sharing of raw data, methodologies, and research tools.

**Navigating the Landscape of Open Linguistic Data Sources:**

While sourcing is crucial, data preservation remains a challenge. Traditional storage methods, such as hard drives, are prone to failures.

* **Repositories:** Established platforms house vast troves of linguistic data. 
To name a few:
  * **Linguistic Data Consortium (LDC):** An institution that curates diverse datasets, encompassing varied linguistic elements from speech patterns to text nuances.
  * **Universal Dependencies Project:** A key resource for those studying grammar and syntax, offering detailed treebank data for numerous languages.
  * **Open Science Framework:** Is a platform designed to support transparent and collective scientific endeavors. 
  Offered as a complimentary, open-source tool by the Center for Open Science, researchers can utilize the OSF to chronicle their projects, team up with peers, and disseminate their data and outcomes. 
  Additionally, researchers can also create preprints accessible to the public even as the project undergoes peer review in a journal.


## Challenges and considerations 

In the linguistic research, there are various challenges and considerations to account for, and among these, a key issue is the durability and accessibility of the data.

1. **Data Quality**: Ensuring that the data collected is accurate, consistent, and complete is essential. **Accuracy** which can refer to pertains the data's veracity, while **consistency** ensures uniformity in data collection and representation. 
**Completeness** guarantees that the data provides a complete view as possible, and no essential details are omitted.
2. **Representation**: With the endless linguistic diversity in the world we life in. 
It becomes necessary to collaborate in every language, minority languages and dialects are adequately represented. 
3. **Ethics**: Protecting participant privacy should also be key point. 
This involves obtaining **consent**, safeguarding privacy, and demonstrating **cultural sensitivity**. 
In the diverse field of linguistics, the type of data collected can span a spectrum from lab-based phonetics experiments to sociolinguistic video interviews. 
Consequently, the inherent characteristics of the data differ substantially. 
For example, data stemming from phonetic experiments can be conveniently de-identified, whereas disseminating video materials from sociolinguistic posses more ethical issues. 
Potential strategies to address these concerns encompass providing textual transcriptions, excluding video content, or procuring formal consents from study participants. 
Crucially, any determinations must attend the importance of safeguarding participants' well-being and confidentiality, all the while adhering to legal prescriptions and the ethical discernment of the researcher.

4. **Standardization**: Here, transcription, and data storage practices come into play. 
These ensure that the data is universally understandable and ensures that the data can be utilized by other researchers for replication or further exploration.

On the topic of **data storage**, an unavoidable dilemma associated with the aging of technology is the failure of storage drives. 
Privately storing data over extended periods is trickier than one might assume. 
Drives can get lost, and even if they do not, hard drive failures are inevitable over time. 
So, how does one safeguard their invaluable linguistic data? One feasible solution is making the data public. 
Not only does this promote data preservation for future reanalysis, but it also encourages transparency in research.

Given the presented variability in the linguistic field, the decision to share data, especially data related to human subjects, warrants careful consideration. 
It becomes vital to assess the kind of study you're conducting: whether it's phonetics, a sociolinguistic interview, or an eye-tracking study. 

As a principal investigator (PI), your objective should be achieving research reproducibility. 
You should as much information as feasible, enabling the broader research community and society to gain a profound understanding of your work.

****

To sum up, open data is important in scientific research for several reasons:

1. Linguistic research is increasingly emphasizing open and accessible data.
2. Data sharing boosts researcher credibility and fosters collaboration.
3. Global initiatives are pushing for greater transparency in sharing.
4. Data preservation is challenging; notable repositories exist to help.
5. Ethical considerations, especially around participant privacy, are crucial.
6. Technological failures necessitate innovative data storage solutions.
7. Decision to share data should consider study type and aim for reproducibility.
